shared/data/tracker_lists/trackersWithParentCompany.json

```diff
@@ -1151,39 +1111,39 @@
             "u": "https://aloodo.com/"
         },
         "amazon-adsystem.com": {
-            "c": "Amazon",
+            "c": "Amazon.com",
```

Comment:
I thought we already updated Amazon.com. Was it just not pushed to the repo?

Reply:
Yeah, it wasn't pushed. I wonder if it's worth just adding this to `.gitignore` if it's regenerated every time you build the extension anyway.

Reply:
Yeah .gitignore works. The amazon change never made it to our backend service when we started using it for the mobile app so it's been `amazon.com` for awhile anyway.